alternator/executor.cc

```diff
@@ -3190,7 +3190,7 @@ future<executor::request_return_type> executor::query(client_state& client_state
                 "Consistent reads are not allowed on global indexes (GSI)"));
     }
     rjson::value* limit_json = rjson::find(request, "Limit");
-    uint32_t limit = limit_json ? limit_json->GetUint64() : query::max_partitions;
+    uint64_t limit = limit_json ? limit_json->GetUint64() : query::partition_max_rows;
```

Comment:
Is this intentional to use different variable here?

Reply:
Actually, it looks like both variables have misleading names - the limit is later used as size of a page to be fetched by the query pager